### DISCUSSION TITLE
Mark as read in feed.

### DISCUSSION THREAD

**Discussion by ccoVeille** [forem/forem#18385] [Feature Requests]
I initiated the discussion on dev.to here

https://dev.to/ccoveille/is-there-a-feature-to-mark-something-as-read-in-devto--3eap

As per @michael-tharrington suggestion, I open a feature request discussion

I would like to get a way to hide posts from the "relevant" filter.

The same way we have a way to archive things in reading list.

I would love to be able to mark as read/hide/archive posts that are in my feed.

The idea is to be able to remove from the feed items I already saw. The reasons are multiple and may vary between users, but I can think about at least the following use cases:

- a post that buzz a lot that stay in the "relevant" page for days
- a post that was already read and user want to hide it, because he added it to his reading list for later
- a post was read or not but is manually hidden because it's irrelevant.
- etc ...

It may be difficult to implement it, I have no idea, but I report as a need without censoring myself.

I report is as a need to hide them from "relevant" section, I let people judge if there is a need to do it in "top" and "latest" ones

Thanks

**Comment by michael-tharrington** [comment]
Thanks so much for sharing this one, Christophe!
